### PR TITLE
Db/datagridcomponent-bindings

### PR DESCRIPTION
**Pull request by sebastianbochan**
### Issues:
- [x] - bindings did not support datagrid
- [x] - datagrid plugin is not initialized (missing constructor)
- [x] - linting
- [x] - cleanup
- [x] - connect to store

### CODE REVIEW

File: ts/masters/modules/dashboard-plugin.src.ts [hunk 1]
```diff
@@ -21,12 +21,17 @@
...
+import DataGridPlugin from '../../Extensions/DashboardPlugins/DataGridPlugin.js';
 
 const G: AnyRecord = Highcharts;
 G.DashboardPlugin = HighchartsPlugin;
 
 if (G.win.Dashboard) {
     HighchartsPlugin.custom.connectHighcharts(Highcharts);
     G.win.Dashboard.PluginHandler.addPlugin(HighchartsPlugin);
+
+    DataGridPlugin.custom.connectDataGrid(DataGrid);
```
[the remark this answers is not in the document]
Comment (bre1470) [reply ts/masters/modules/dashboard-plugin.src.ts:35]: This is just for classic mode. This part is not for ESM-based demos.

File: ts/masters/modules/dashboard-plugin.src.ts [hunk 2]
```diff
@@ -21,12 +21,17 @@
...
 
 import Highcharts from '../../Core/Globals.js';
+import DataGrid from '../../DataGrid/DataGrid.js';
 import HighchartsPlugin from '../../Extensions/DashboardPlugins/HighchartsPlugin.js';
+import DataGridPlugin from '../../Extensions/DashboardPlugins/DataGridPlugin.js';
 
 const G: AnyRecord = Highcharts;
 G.DashboardPlugin = HighchartsPlugin;
 
 if (G.win.Dashboard) {
```
[the remark this answers is not in the document]
Comment (bre1470) [reply ts/masters/modules/dashboard-plugin.src.ts:31]: In ESM you have to do the connection yourself.

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

**Review by karolkolodziej — APPROVED** [review]
This part looks good! In the future, this demo needs some cleaning :)

File: ts/Dashboard/Actions/Bindings.ts [hunk 3]
```diff
@@ -111,12 +123,17 @@ class Bindings {
             case 'HTML':
                 component = HTMLComponent.fromJSON(json as HTMLComponent.ClassJSON);
                 break;
-            default:
+            case 'Highcharts': 
                 const componentClass = Component.getComponent(json.$class);
```
Comment (pawellysy) [inline ts/Dashboard/Actions/Bindings.ts:127]: If this is only for `Highcharts` Component, there is no need for selecting the correct component, we can define it straight away.

**Review by pawellysy — CHANGES_REQUESTED** [review]
Few comments

**Review by pawellysy — APPROVED** [review]
Looks good!

**Pull request merged**